src/src/tls-gnu.c

```diff
@@ -13,7 +13,7 @@ tls.c when USE_GNUTLS has been set.
 
 The code herein is a revamp of GnuTLS integration using the current APIs; the
 original tls-gnu.c was based on a patch which was contributed by Nikos
-Mavroyanopoulos.  The revamp is partially a rewrite, partially cut&paste as
+Mavrogiannopoulos.  The revamp is partially a rewrite, partially cut&paste as
```

Comment:
https://pear.php.net/user/nmav is a place using the other spelling, and I strongly suspect that the English author/maintainer of Exim just copy/pasted it from the email at the time.  I _think_ this author has changed romanisation from the Greek canonical form.  About to send the email to check.